app/classes/document_query.rb

```diff
@@ -115,6 +115,7 @@ def suggestion_highlight
     }
   end
 
+  # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
```

Comment:
Disable rubocop check for this function as this block of code is using the elasticsearch DSL. Splitting it up will makes the code harder to read IMO.